pkgs/applications/misc/smcipmitool/default.nix

```
@@ -0,0 +1,68 @@
+{ lib, stdenv
+, fetchurl
+, makeWrapper
+, patchelf
+, fontconfig
+, freetype
+, gcc
+, gcc-unwrapped
+, iputils
+, psmisc
+, xorg }:
+
+stdenv.mkDerivation rec {
+  pname = "SMCIPMItool";
+  version = "2.27.2";
+  buildVersion = "230221";
+
+  src = fetchurl {
+    url = "https://www.supermicro.com/wdl/utility/SMCIPMItool/Linux/SMCIPMITool_${version}_build.${buildVersion}_bundleJRE_Linux_x64.tar.gz";
+    hash = "sha256-DL39p3sm72DtxDKPV7aDwfP66Yj1nJ7rafvK/Oj3t8I=";
+  };
+
+  nativeBuildInputs = [ patchelf makeWrapper ];
+  buildPhase = with xorg;
```

```suggestion
  buildPhase =
```

Avoid `with` with all costs.